app/_hub/kong-inc/ai-proxy/overview/_index.md

```diff
@@ -6,6 +6,7 @@ The AI Proxy plugin lets you transform and proxy requests to a number of AI prov
 
 The plugin accepts requests in one of a few defined and standardised formats, translates them to the configured target format, and then transforms the response back into a standard format.
 
+{% if_version gte:3.7.x %}
```

Comment:
```suggestion
{% if_version lte:3.6.x %}
The AI Proxy plugin supports `llm/v1/chat` and `llm/v1/completion` style requests for all of the following providers:
* OpenAI
* Cohere
* Azure
* Anthropic
* Mistral (raw and OLLAMA formats)
* Llama2 (raw, OLLAMA, and OpenAI formats)
{% endif_version %}
{% if_version gte:3.7.x %}
```
My bad for not catching the conditional rendering! I did remove this bit that I added as a suggestion when I added the table, maybe we'll want to add it in for versions before 3.7?